src/lib/components/ui/FileInputField/README.mdx

```diff
@@ -0,0 +1,215 @@
+---
+name: FileInputField
+menu: 'UI'
+route: /components/ui/file-input-field
+---
+
+# FileInputField
+
+FileInputField allows users to select files to upload.
+
+import {
+  Playground,
+  Props,
+} from 'docz'
+import { FileInputField } from './FileInputField'
+
+## Basic Usage
+
+To implement the FileInputField component, you need to import it first:
+
+```js
+import { FileInputField } from '@react-ui-org/react-ui';
+```
+
+And use it:
+
+<Playground>
+  <FileInputField id="my-file-input-field" label="Attachment" />
+</Playground>
+
+See [API](#api) for all available options.
+
+## General Guidelines
+
+- Apply [native HTML attributes](#forwarding-html-attributes) to control **what
+  and how many files** users can select.
+
+- Use **short and descriptive labels**, ideally nouns rather than seemingly
+  polite phrases like _Please attach your file here_. Short labels will help
+  your users accomplish their task faster.
+
+- Only make the FileInputField's label invisible when there is **another visual
+  clue** to guide users through filling the input.
+
+- When a short label is not enough, use **help texts to guide users** before
+  they enter anything.
+
+- Use **clear, calm error messages** when there's a problem with what they
+  entered. Be positive and focus on solutions to make the error message helpful.
+
+👉 FileInputField implements native HTML `input type="file"`. Therefore, it is
+not possible to affect neither its translations (they depend on a browser), nor
+appearance.
+[Custom translations and appearance](https://github.com/react-ui-org/react-ui/issues/244)
+are considered to be implemented at some point in the future. However, from the
+layout perspective, FileInputFields work just like any other form fields.
+
+## Sizes
+
+Full-width fields span the full width of a parent:
+
+<Playground>
+  <FileInputField
+    id="my-file-input-field-full-width"
+    fullWidth
+    label="First name"
+  />
+</Playground>
+
+## Forwarding HTML Attributes
+
+When you want to refine your file input fields even further, you
+can **add whatever HTML attribute you like.** All attributes that don't
+interfere with [component's API](#api) are forwarded to the native HTML input.
+
+<Playground>
+  <FileInputField
+    accept=".pdf,.jpg,.jpeg,.png"
+    helpText={`
+      Choose up to 10 files. Allowed extensions are .pdf, .jpg, .jpeg, or .png.
+      Size limit is 10 MB.
+    `}
+    id="my-file-input-field-forwarding"
+    label="Attachment"
+    multiple
+  />
+</Playground>
+
+👉 Refer to the MDN reference for the full list of supported attributes of the
+[file input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file)
+type.
+
+## Invisible Label
+
+In some cases, it may be convenient to visually hide the field label. The label
+remains accessible to assistive technologies.
+
+While it may be acceptable for login screens with just a few fields or for other
```

Comment:
It does not seem to me right to talk about Login screen when we are documenting FileInput